# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1329586974 18000
#      Sat Feb 18 12:42:54 2012 -0500
# Node ID e2306a8e054c2676510a02a6253ff316545b742c
# Parent  688e19ca262ba9560b1e1ab86ba7457bd11e2471
Revert changeset 688e19ca262b due to regression.
See bug # 34595.

diff --git a/scripts/plot/print.m b/scripts/plot/print.m
--- a/scripts/plot/print.m
+++ b/scripts/plot/print.m
@@ -388,20 +388,16 @@ function print (varargin)
     endif
 
     ## call the graphcis toolkit print script
     switch (get (opts.figure, "__graphics_toolkit__"))
     case "gnuplot"
       opts = __gnuplot_print__ (opts);
     otherwise
       opts = __fltk_print__ (opts);
-      if (isunix ())
-        ## FIXME - Is there a way to explicitly determine the pid numbers?
-        waitpid (0);
-      endif
     endswitch
 
   unwind_protect_cleanup
     ## restore modified properties
     if (isstruct (props))
       for n = numel(props):-1:1
         if (ishandle (props(n).h))
           set (props(n).h, props(n).name, props(n).value{1});
